worlds/sc2/item_descriptions.py

```diff
@@ -506,7 +506,10 @@ def _ability_desc(unit_name_plural: str, ability_name: str, ability_description:
     item_names.HIGH_TEMPLAR: "Potent psionic master. Can use the Feedback and Psionic Storm abilities. Can merge into an Archon.",
     item_names.DARK_TEMPLAR: "Deadly warrior-assassin. Permanently cloaked. Can use the Shadow Fury ability.",
     item_names.IMMORTAL: "Assault strider. Can use Barrier to absorb damage.",
-    item_names.COLOSSUS: "Battle strider with a powerful area attack. Can walk up and down cliffs. Attacks set fire to the ground, dealing extra damage to enemies over time.",
+    item_names.COLOSSUS: inspect.cleandoc("""
+        Battle strider with a powerful area attack. Can walk up and down cliffs.
+        Attacks set fire to the ground, dealing extra damage to enemies over time.
+    """),

```

Comment:
These multi-line description changes can be reverted now that there's auto-newlining on periods.

Reply:
Done.